competition-simulator/programming.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Programming
+---
+
+# Programming
+
+## Developing your code
+
+On first run, the robot will execute an example program for convenience. On first run, this will be copied to the directory `competition-simulator` is stored in:
```

Comment:
```suggestion
On first run, the robot will execute an example program for convenience. On first run, this will be copied to the directory `competition-simulator-<version>` is stored in:
```